doc/news/program_capture_sharp_bits.rst

```diff
@@ -99,9 +99,9 @@ Array([0.99875027+0.j        , 0.        +0.j        ,
 Gradients
 ---------
 
-Currently the devices ``default.qubit`` and ``lightning.qubit`` are the only devices
-that support gradients with program capture enabled. ``default.qubit`` currently supports
-``adjoint``, ``backprop`` and ``finite-diff``. ``lightning.qubit`` currently only supports 
+Currently the devices ``default.qubit``, ``lightning.qubit``, ``lightning.kokkos``, and ``lightning.gpu`` 
+are the only devices that support gradients with program capture enabled. ``default.qubit`` currently 
+supports ``adjoint``, ``backprop`` and ``finite-diff``. ``lightning.qubit`` currently only supports 
```

Comment:
LK and LGPU also have support for adjoint and paramshift, don't they?

```suggestion
supports ``adjoint``, ``backprop`` and ``finite-diff``. ``lightning.qubit``, ``lightning.kokkos``, and ``lightning.gpu`` currently only supports 
```

Reply:
technically finite diff too, but that's not really a "device" derivative anyway.